### PR TITLE
Bugfix/update data archive format

### PR DESCRIPTION
**Pull request by liujoshua**

### CODE REVIEW

**Review by dephillipsmichael — APPROVED** [review]
Looks good

**Comment by liujoshua** [comment]
Depends on https://github.com/ResearchStack/ResearchStack/pull/449

**Comment by dephillipsmichael** [comment]
Looks good

File: researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/factory/ArchiveFileFactory.java [hunk 1]
```diff
@@ -127,6 +126,7 @@ public SurveyAnswer surveyAnswer(StepResult stepResult) {
                 answer = new SurveyAnswer.ChoiceSurveyAnswer(stepResult);
                 break;
             case Integer:
+            case Decimal:
```
Comment (dephillipsmichael) [inline researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/factory/ArchiveFileFactory.java:129]: I was hitting this crash too, thanks for fixing.

**Pull request merged**